.github/pull_request_template.md

```diff
@@ -0,0 +1,5 @@
+Please ensure this checklist is followed for any pull requests for this repo. This checklist must be checked by both the PR creator and by anyone who reviews the PR.
+* [ ] Relevant documentation for this PR has to be completed and reviewed by the documentation team (@ioptio, @mdr0id, or @Eirik0) before the PR can be merged
```

Comment:
A valid parsing of this sentence is that the documentation has to be completed **by** the documentation team, rather than the PR author. Probably doesn't matter though, since the important part is that the documentation is completed 😄